requirements.txt

```diff
@@ -11,4 +11,5 @@ pyunpack==0.2.2
 scikit-learn==0.23.2
 scikit-optimize==0.8.1
 seaborn==0.11.0
-tensorflow==2.3.1
\ No newline at end of file
+tensorflow==2.4.0
```

Comment:
I am not sure why you changed the tensorflow version. Did you find a bug with 2.3.1?

Reply:
Checked it. The warning is due to TF 2.3, we should use 2.4, the default one of Colab.